modules/ROOT/pages/upgrade-index.adoc

```diff
@@ -286,7 +286,23 @@ Refer to https://help.mulesoft.com/s/article/How-to-Troubleshoot-RTF-Appliance-U
 
 == Roll Back an Appliance Upgrade
 
-If you cannot resolve issues with an upgrade, you can roll back the upgrade steps that executed.
+If you cannot resolve issues with an upgrade, you can roll back the upgrade using one of the following options:
```

Comment:
Does "with an upgrade" mean "by upgrading" or "that occurred as a result of an upgrade"?